.gitignore

```
@@ -25,4 +25,7 @@ go.work
 .idea/
 
 # Built files
-dist/
\ No newline at end of file
+dist/
+
+# Mock source files
+/mocks
```

```suggestion
/mocks

```
Just missing a new line at the end.